semgrep/dependencyparser/models.py

```diff
@@ -24,7 +23,7 @@ class PackageManagers(Enum):
     MAVEN = "maven"
 
 
-@frozen(eq=True, order=True)
+@dataclass(eq=True, order=True)
```

Comment:
This `dataclass` change was to make JSON serialization possible without doing it manually